lib/workers/repository/onboarding/branch/index.spec.ts

```diff
@@ -71,8 +71,10 @@ describe('workers/repository/onboarding/branch/index', () => {
       git.getFileList.mockResolvedValue(['package.json']);
       fs.readLocalFile.mockResolvedValue('{}');
       await checkOnboardingBranch(config);
+      const file = git.commitFiles.mock.calls[0][0].files[0];
+      expect(file.type).toBe('addition');
       const contents =
-        git.commitFiles.mock.calls[0][0].files[0].contents?.toString();
+        file.type === 'addition' ? file.contents?.toString() : null;
```

Comment:
Can't we simply assert it, as it must be an addition?

Reply:
It turns out, `assert(file.type === 'addition');` doesn't imply further type narrowing, so we still need to check the type.  Or did you mean something else? 🤔

Reply:
see above

Reply:
Get it 😅